pex/scie/__init__.py

```diff
@@ -39,6 +42,7 @@ def register_options(parser):
 
     parser.add_argument(
         "--scie",
+        "--par",
```

Comment:
As the issues attached to this PR prove, people in the world know "PAR"; so it seems to make sense to add a `--par` alias to this one option for discoverability by those people. If they need more than the default `--par` treatment, they really must learn about scies and `--scie-*` advanced options anyhow.